static/modules/buttons.js

```diff
@@ -106,8 +107,11 @@ export const buttons = {
 
   primary: new Button(" ", 4),
   secondary: new Button("e", 5),
-  fullscreen: new Button("f", 8),
+  select: new Button(" ", 0),
+  back: new Button("Tab", 1),
```

Comment:
Escape seems the most natural, but you can't use it since it takes you out of fullscreen (there's no way to prevent this for security reasons). Shift+Enter is a little janky and would require either an idiosyncrasy or rewrite of a lot of `buttons.js` to accomodate detecting modifier keys.